01_P/P_2_0_01/sketch.js

```diff
@@ -18,23 +18,23 @@
 
 /**
  * drawing a filled circle with lines.
- *   
+ *
  * MOUSE
  * position x          : length
  * position y          : thickness and number of lines
- * 
+ *
  * KEYS
  * s                   : save png
  */
- 
+'use strict';
+
 function setup() {
   createCanvas(550, 550);
+  strokeCap(SQUARE);
```

Comment:
Moved `strokeCap` and `noFill` to the `setup` since they only have to be used once. Also removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).